cogstat/cogstat_stat.py

```diff
@@ -699,6 +699,61 @@ def variable_pair_standard_effect_size(data, meas_lev, sample=True, normality=No
                                                                                  classes="table_cs_pd"))
     return standardized_effect_size_result
 
+def correlation_matrix(data, x):
+    """Create Pearson's correlation correlation matrix for assessment of multicollinearity.
+
+    Parameters
+    ----------
+    data : pandas dataframe
+    x : list of str
+        list of explanatory variable names
+
+    Returns
+    -------
+    html text
+    """
+
+    corr_table = data[x].corr()
+    table = _("Pearson correlation matrix of explanatory variables:")
+    table += _format_html_table(corr_table.to_html(bold_rows=False, escape=False, classes="table_cs_pd")) + "\n"
+    return table
+
+def vif_table(data, x):
```

Comment:
please use a more explanatory name for x

Reply:
Variable names updated.